### PR TITLE
(refactor) added custom props to support loadingIndicator, background colours

### LINKED ISSUE #2: Custom props to support
Need custom props to support Webview Loading Indicator and customise the indicator colour, background colour etc

### PR DESCRIPTION
**Pull request by justinkx**
closes #2 
**Custom Props Added**

1. **showLoading** - Prop to show loading indicator when WebView loads hCaptcha content
2. **backgroundColor** - Background colour for the SafeAreaView and WebView backdrop so as to customise for apps having specific theme on the page
3. **loadingIndicatorColor** - Color for the ActivityLoadingIndicator of WebView **renderLoading**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,62 +1,96 @@
...
+  state = {
+    show: false,
+  };
+  show = () => {
+    this.setState({ show: true });
+  };
+  hide = () => {
+    const { onMessage } = this.props;
+    this.setState({ show: false });
+    onMessage({ nativeEvent: { data: 'cancel' } });
```
Comment (justinkx) [inline index.js:19]: Close hCaptcha on clicking android Hardware back button

**Comment by e271828-** [comment]
@justinkx Thanks! Would you mind adding a test?

File: index.js (hunk 1 above)
Comment (e271828-) [reply index.js:19]: Please add as inline comment.

**Comment by e271828-** [comment]
@justinkx appreciate it! Forked this to add a github action and merged as #3 .

**Pull request closed without merging**